frontend/src/components/dialogs/SecretDialogWrapper.vue

```diff
@@ -63,7 +67,12 @@ export default {
           return 'GcpDialog'
         default: {
           const name = upperFirst(head(split(this.visibleDialog, '-')))
-          return `${name}Dialog`
+          const componentName = `${name}Dialog`
```

Comment:
```js
          const name = upperFirst(camelCase(this.visibleDialog))
```
The line above

Reply:
This does not work as we need to get only the first part of the dialog name